lib/bundler.js

```diff
@@ -169,13 +180,12 @@ module.exports = async function bundler({
         //each bundle's dependencies can have a paths.json whose keys should be excludes
         if (Array.isArray(dependencies)) {
             await Promise.all(dependencies.map(dependency => {
-                try {
-                    const dependencyPaths = require(`${rootPath}/${dependency}/views/build/grunt/paths.json`);
+                const dependencyPaths = loadExtensionPaths(dependency);
+                if (Object.keys(dependencyPaths).length) {
```

Comment:
`Object.keys(dependencyPaths)` should be a variable, because you use it multiple times.